apps/language_server/lib/language_server/providers/execute_command/manipulate_pipes/ast.ex

```diff
@@ -64,4 +70,77 @@ defmodule ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipes.AST d
   defp do_to_pipe(node, acc) do
     {node, acc}
   end
+
+  def ast_to_string(ast) do
+    Macro.to_string(ast, fn node, rendered ->
+      case sigil_call(node, fn _a, s -> s end) do
+        {:ok, parsed_sigil} -> parsed_sigil
+        _ -> rendered
+      end
+    end)
+  end
+
+  # The code below is copied from the Elixir source-code because a
+  # fix which was introduced in
+  # https://github.com/elixir-lang/elixir/commit/88d82f059756ed1eb56a562fae7092f77d941de8
+  # is needed for proper treatment of sigils in our use-case.
+  defp sigil_call({sigil, meta, [{:<<>>, _, _} = parts, args]} = ast, fun)
```

Comment:
add a TODO with info which part may be removed when we require elixir 1.13 (or 1.12 if the fix gets backported)